src/core.js

```diff
@@ -432,17 +430,20 @@ function(i, name) {
 });
 
 function isArraylike( obj ) {
+	var type, length;
+
+	if ( obj == null ||
+		( type = jQuery.type( obj ) ) === "function" ||
+		type === "number" ||
+		jQuery.isWindow( obj ) ) {
+		return false;
```

Comment:
Here the code said `false` for `null`, `undefined`, `"number"`, `NaN`, `"function"` only.